realm/util.gno

```diff
@@ -21,6 +21,30 @@ func getUserPosts(userAddr std.Address) *UserPosts {
 	return userPosts.(*UserPosts)
 }
 
+// Get the UserPosts for the userAddr. If not found, add a new UserPosts to
+// gUserPostsByAddress and update gUserAddressByName with the username.
+// (The caller usually has already called usernameOf to get the username, but if
+// it is "" then this will get it.)
```

Comment:
Don't understand this comment :)

Reply:
This function has an argument for the user address, but also an argument for the username. We want to save CPU cycles. In all the cases where code needs to call this function, it has already looked up the user name by user address.
https://github.com/gnolang/gnosocial/blob/1c86452df7b8f9400863386b98edeaa3c7c12aa9/realm/public.gno#L70
So it should not be necessary for this function to look it up again. The comment is justifying why this function has a username argument. For completeness, this function will look up the username by user address, but only if the caller doesn't provide the username.

Reply:
Do you think that the comment needs to be updated?

Reply:
In the next PR ;)